api-catalog-services/src/test/java/org/zowe/apiml/apicatalog/services/status/LocalApiDocServiceTest.java

```diff
@@ -67,37 +62,41 @@ class LocalApiDocServiceTest {
 
     @Mock
     private CloseableHttpClient httpClient;
-    private CloseableHttpResponse response;
 
     @Mock
     private InstanceRetrievalService instanceRetrievalService;
 
     private APIDocRetrievalService apiDocRetrievalService;
 
-    @BeforeEach
-    void setup() throws IOException {
-        response = mock(CloseableHttpResponse.class);
-        when(response.getCode()).thenReturn(HttpStatus.SC_OK);
-        when(httpClient.execute(any())).thenReturn(response);
+    @Mock
+    private ApimlLogger apimlLogger;
 
+    @BeforeEach
+    void setup() {
         GatewayClient gatewayClient = new GatewayClient(getProperties());
         apiDocRetrievalService = new APIDocRetrievalService(
             httpClient,
             instanceRetrievalService,
             gatewayClient);
+
+        ReflectionUtils.doWithFields(apiDocRetrievalService.getClass(), field -> {
+            if (field.getAnnotation(InjectApimlLogger.class) != null) {
+                ReflectionUtils.makeAccessible(field);
+                field.set(apiDocRetrievalService, apimlLogger);
+            }
+        });
```

Comment:
Nothing bad with this, but in the project we normally go with the `ReflectionTestUtils` helper class from Spring
```suggestion
        ReflectionTestUtils.setField(apiDocRetrievalService, "apimlLogger", apimlLogger);
```